server/api.js

```diff
@@ -1,59 +1,39 @@
 "use strict"
 
+const denodeify = require("denodeify")
```

Comment:
Or would [`util.promisify`](https://nodejs.org/en/blog/release/v8.0.0/#improved-support-for-promises) work?